SPECS/iptables/ip4save

```diff
@@ -2,12 +2,12 @@
 *filter
 :INPUT DROP [0:0]
 :FORWARD DROP [0:0]
-:OUTPUT DROP [0:0]
+:OUTPUT ACCEPT [0:0]
```

Comment:
To be honest, I like the idea of starting the config with a set of deny-all rules and then enabling only what's needed. I understand that in this case we end up accepting all output later on, but now we're introducing an inconsistency in the config for the sake of saving one line but at a cost of potentially allowing too much output if we ever want to change this policy at some point in the future.

Reply:
Apologies forgot to reply to this comment. From an ops point of view, setting the policy as DROP as in
iptables -P OUTPUT DROP 
can be a little problematic. If someone logs into the machine and does a iptables -F, it will clear all the rules, and the default policy is to close the machine down, and you would completely lose access.

If it is set to ACCEPT as a default policy, the machine would fail open, allowing access. So I guess it is less secure. 

From my point of view, most places have security by depth. So the tradeoff of it failing open is better from an operational point of view.

In addition, I would prefer to have the default policy on my chains as ACCEPT, and have explicit rules to drop traffic but via a REJECT as in
iptables -A [INPUT|OUTPUT] -j REJECT

This way if debugging a network issue, you would still get the TCP resets from your server. This is immensely useful to help debug server access issues.


Let me know, I am happy to set this to DROP

Reply:
We had a similar discussion a few years ago related to this PR incase it is helpful https://github.com/microsoft/CBL-Mariner/pull/229